### PR TITLE
[irods/irods#5249] add iadmin get_delay_server_info subcommand (main)

### PR DESCRIPTION
**Pull request by korydraughn**
This PR builds upon the work of #262

### CODE REVIEW

File: src/iadmin.cpp [hunk 1]
```diff
@@ -2387,6 +2460,24 @@ usage( char *subOpt ) {
...
 
+    char* get_delay_server_info_usage[] = {
+        "get_delay_server_info",
+        " ",
+        "Prints information about the delay server as JSON.",
+        " ",
+        "This command allows administrators to identify which server is running the",
+        "delay server and if the delay server is being migrated.",
+        " ",
+        "This information is retrieved from R_GRID_CONFIGURATION.",
```
Comment (trel) [inline src/iadmin.cpp:2471]: "the R_GRID_CONFIGURATION table."

?

not sure it's necessary, just might be clarifying for a new reader.

File: src/iadmin.cpp [hunk 2]
```diff
@@ -143,6 +149,68 @@ auto get_replica_value(
...
+            return CAT_INSUFFICIENT_PRIVILEGE_LEVEL;
+        }
+    }
+    catch (const irods::exception& e) {
+        std::cerr << "ERROR: " << e.client_display_what() << '\n'; // TODO Is there a common format for printing error messages?
+        return e.code();
+    }
+    catch (const std::exception& e) {
+        std::cerr << "ERROR: " << e.what() << '\n';
+        return 1; // TODO What error code should we use here?
```
Comment (trel) [inline src/iadmin.cpp:173]: seems okay

File: src/iadmin.cpp (hunk 1 above)
Comment (korydraughn) [reply src/iadmin.cpp:2471]: I actually had that originally lol, but tried to match set_delay_server's help text.

Will change to what you've recommended.

File: src/iadmin.cpp (hunk 2 above)
Comment (korydraughn) [reply src/iadmin.cpp:173]: Yeah, I guess this is correct. The OS doesn't know what the iRODS error code is. We can print the iRODS error code to the terminal and return a non-zero value for these cases.

File: src/iadmin.cpp (hunk 1 above)
Comment (trel) [reply src/iadmin.cpp:2471]: nice - right back on me.     i'm sure i went with brevity at the time.   let's use more words.

**Comment by korydraughn** [comment]
This PR is dependent upon https://github.com/irods/irods/pull/6272

Tests for this PR will be added following the merging of the irods/irods PR.

**Review by alanking — APPROVED** [review]

**Pull request merged**